res/values/strings.xml

```diff
@@ -241,6 +241,15 @@
 	<string name="ssl_validator_label_signature_algorithm">Algorithm:</string>
 	<string name="ssl_validator_null_cert">The certificate could not be shown.</string>
 	<string name="ssl_validator_no_info_about_error">- No information about the error</string>
+
+    <!-- QUOTA ACTIVITY STRINGS -->
+    <string name="title_activity_quota_display">QuotaDisplayActivity</string>
```

Comment:
This title is not user-friendly. Anyway, maybe it can be just removed. It's set only for the ```label``` property in the Manifest, and this activity can't be opened from other apps, so it won't ever be printed on screen. Or am I losing something?